next-env.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference types="next" />
+/// <reference types="next/types/global" />
```

Comment:
I tend to not leave commented out code

Reply:
This is an automated type reference file from next, so TS will look for and apply these types globally. Weird syntax I know, you can read more about it here https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html